Form-Controls/index.html

```diff
@@ -12,16 +12,48 @@
       <h1>Product Pick</h1>
     </header>
     <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <form action="/submit-form" method="post">
```

Comment:
I don't think your project has /submit-form url. If not, I would not recommend adding it. For the sake of your exercise you can use action="#" which is a convention to say "we don't have were to send this form yet".